modular_skyrat/modules/science_tools/tool_designs.dm

```diff
@@ -0,0 +1,69 @@
+/datum/design/jawsoflife/science
+	id = SCIENCE_JAWS_OF_LIFE_DESIGN_ID // added one more requirment since the Jaws of Life are a bit OP
+	build_path = /obj/item/crowbar/power/science
+	departmental_flags = DEPARTMENT_BITFLAG_SCIENCE
+
+/datum/design/jawsoflife/science/New()
+	name = ("Science " + name)
+	desc += " but lacking the door-opening power"
+
+	return ..()
+
+/datum/design/handdrill/science
+	id = SCIENCE_DRILL_DESIGN_ID
+	build_type = PROTOLATHE | AWAY_LATHE
+	build_path = /obj/item/screwdriver/power/science
+	departmental_flags = DEPARTMENT_BITFLAG_SCIENCE
+
+/datum/design/handdrill/science/New()
+	name = ("Science " + name)
+	desc += " with a science paintjob"
+
+	return ..()
+
+/datum/design/laserscalpel
+	construction_time = 1 SECONDS
+
+/datum/design/mechanicalpinches
+	construction_time = 1 SECONDS
+
+/datum/design/searingtool
+	construction_time = 1 SECONDS
+
+/datum/design/healthanalyzer
+	construction_time = 5 SECONDS
+
+/datum/design/healthanalyzer_advanced
+	construction_time = 5 SECONDS
+
+/datum/design/laserscalpel/New()
+	build_type |= MECHFAB
+	category += (RND_CATEGORY_MECHFAB_EQUIPMENT + RND_SUBCATEGORY_MECHFAB_EQUIPMENT_MEDICAL)
+
+	return ..()
+
+/datum/design/mechanicalpinches/New()
+	build_type |= MECHFAB
+	category += (RND_CATEGORY_MECHFAB_EQUIPMENT + RND_SUBCATEGORY_MECHFAB_EQUIPMENT_MEDICAL)
+
+	return ..()
+
+/datum/design/searingtool/New()
+	build_type |= MECHFAB
+	category += (RND_CATEGORY_MECHFAB_EQUIPMENT + RND_SUBCATEGORY_MECHFAB_EQUIPMENT_MEDICAL)
+
+	return ..()
+
+// its fine to give them health analyzers, the choke for medibot production is the medkits
+/datum/design/healthanalyzer/New()
+	build_type |= MECHFAB
+	category += (RND_CATEGORY_MECHFAB_EQUIPMENT + RND_SUBCATEGORY_MECHFAB_EQUIPMENT_MEDICAL)
+
+	return ..()
+
+
+/datum/design/healthanalyzer_advanced/New()
+	build_type |= MECHFAB
+	category += (RND_CATEGORY_MECHFAB_EQUIPMENT + RND_SUBCATEGORY_MECHFAB_EQUIPMENT_MEDICAL)
+
+	return ..()
```

Comment:
Why are you opting to put this in the mechfab opposed to the science protolathe? There isn't any other standard tool item, aside form the scryer, that can be printed from the mechfab.

Reply:
It was agreed apon in dev channels that this would be fine as long as the tools only robo need would be put in the exofab, as that tells scientists they arent intended to have these things

Reply:
What is inherently wrong with scientists having access to surgery tools? If anything it would encourage scientists to do more varied experiments.

Reply:
resolved(?) in discord, golden doesnt want them outside fabs